plugins/meson.build

```diff
@@ -1,3 +1,4 @@
+subdir('cpu')
```

Comment:
maybe `cpu-microcode`? I'm not sure. Maybe `cpu` is best?

Reply:
I went back and forth and flipped a coin.  Whichever you prefer.

Reply:
Leave it as it, no worries.